sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy

```diff
@@ -143,11 +141,15 @@ class APISpec extends Specification {
         alternateCredential = getCredential(SECONDARY_STORAGE)
         blobCredential = getCredential(BLOB_STORAGE)
         premiumCredential = getCredential(PREMIUM_STORAGE)
+        Properties props = System.getProperties()
+        props.setProperty("reactor.bufferSize.x", "16")
+        props.setProperty("reactor.bufferSize.small", "100")
```

Comment:
We may want to store off the configured value and reset it on completion of the test run.